pkg/apis/options/options.go

```diff
@@ -110,6 +110,8 @@ type Options struct {
 	PubJWKURL       string `flag:"pubjwk-url" cfg:"pubjwk_url"`
 	GCPHealthChecks bool   `flag:"gcp-healthchecks" cfg:"gcp_healthchecks"`
 
+	CallbackErrorRedirects []string `flag:"callback-error-redirect" cfg:"callback-error-redirects"`
```

Comment:
`cfg` uses underscores instead of hyphens
```suggestion
	CallbackErrorRedirects []string `flag:"callback-error-redirect" cfg:"callback_error_redirects"`
```